static/templates/copy_code_button.hbs

```diff
@@ -0,0 +1,5 @@
+<button class="btn pull-left copy_codeblock" data-toggle="tooltip" title="{{t "Copy code" }}" aria-label="{{t "Copy code" }}">
+    <svg height="17" width="14" viewBox="0 0 1000 1000" xmlns="http://www.w3.org/2000/svg" id="clipboard_image">
+        <path fill="#777" d="M128 768h256v64H128v-64z m320-384H128v64h320v-64z m128 192V448L384 640l192 192V704h320V576H576z m-288-64H128v64h160v-64zM128 704h160v-64H128v64z m576 64h64v128c-1 18-7 33-19 45s-27 18-45 19H64c-35 0-64-29-64-64V192c0-35 29-64 64-64h192C256 57 313 0 384 0s128 57 128 128h192c35 0 64 29 64 64v320h-64V320H64v576h640V768zM128 256h512c0-35-29-64-64-64h-64c-35 0-64-29-64-64s-29-64-64-64-64 29-64 64-29 64-64 64h-64c-35 0-64 29-64 64z" />
+    </svg>
```

Comment:
Does this SVG come from some third-party source?  If so we should attribute it.

Reply:
Also, ideally we'd use the same SVG that we use for the "View Source" copy button; maybe extract and reuse that?

Reply:
I see https://github.com/zulip/zulip/pull/16179/files#r474816651; I think I agree this one looks better.  Want to do a follow-up PR for this?